core/drivers/clk/clk-stm32mp13.c

```diff
@@ -1487,7 +1487,7 @@ static int stm32_clk_parse_fdt_all_oscillator(const void *fdt,
 		fdt_err = clk_stm32_parse_oscillator_fdt(fdt, osc_node,
 							 osc_data->name, osci);
 		if (fdt_err < 0)
```

Comment:
I think it should still panic if <0 and != FDT_ERR_NOTFOUND

Reply:
Oh at least return the error code :)

Reply:
right, i'll panic if case of an unexpected error code.